packages/bull-vault/src/AuctionBull.sol

```diff
@@ -523,8 +526,11 @@ contract AuctionBull is UniFlash, Ownable, EIP712 {
         } else {
             // WETH to take out of Euler
             uint256 wethFromEuler = wethInCollateral.sub(_params.wethTargetInEuler);
+            console.log('wethFromEuler', wethFromEuler);
+            console.log('wethNeededForCrab', ethNeededForCrab);
```

Comment:
I think there's a missing case here where wethFromEuler > ethNeededForCrab so we can't reach the branch.